hack/build/docker/cdi-func-test-file-host-http/nginx.conf

```diff
@@ -0,0 +1,25 @@
+worker_processes 1;
```

Comment:
how much of this is a dup of pr #328

Reply:
I'll just leverage whatever's in this pr for #328. We'll merge this pr first and then i'll rebase.

The nginx config as it is now should be fine. All I care about is that the tinyCore.iso is present in the /tmp/shared/ directory so it will be served by the webserver.

Reply:
Okay, sounds good.  I'll get the template fixed up and the build flow cleaned up today and it should be good to go.